src/components/DataDimension/FormulaItem.js

```diff
@@ -0,0 +1,197 @@
+                        scaleX: 1,
+                        scaleY: 1,
+                    }),
+              transition,
+          }
+        : undefined
+
+    let insertPosition = undefined
+    if (over?.id === id) {
+        // This item is being hovered over by the item being dragged
+        if (activeIndex === -1) {
+            //The item being dragged came from the expression options
+            // so we will insert after
+            insertPosition = AFTER
+        } else {
+            // The item being dragged is being moved in the formula
+            // so if the item is before the item being dragged, use the
+            // BEFORE position. Otherwise use the AFTER position
+            insertPosition = index > activeIndex ? AFTER : BEFORE
+        }
+    } else if (isLast && over?.id === LAST_DROPZONE_ID) {
+        insertPosition = AFTER
+    }
+
+    const handleClick = (e) => {
+        const tagname = e.target.tagName
+        clearTimeout(clickTimeoutId)
+        const to = setTimeout(function () {
+            if (tagname !== 'INPUT') {
+                onClick(id)
+            } else {
+                inputRef.current && inputRef.current.focus()
+            }
+        }, maxMsBetweenClicks)
+        setClickTimeoutId(to)
+    }
+
+    const handleDoubleClick = () => {
+        clearTimeout(clickTimeoutId)
+        setClickTimeoutId(null)
+        onDoubleClick(id)
+    }
+
+    const handleChange = (e) => onChange({ index, value: e.target.value })
+
+    const getContent = () => {
+        if (type === TYPE_INPUT) {
+            return (
+                <>
+                    {DragHandleIcon}
+                    <span className="input-positioner">
+                        <span className="width-machine" aria-hidden="true">
+                            {value}
+                        </span>
+                        <input
+                            id={id}
+                            name={label}
+                            onChange={handleChange}
+                            value={value}
+                            type="number"
+                            ref={inputRef}
+                        />
+                    </span>
+                    <style jsx>{styles}</style>
+                </>
+            )
+        }
+
+        if (type === TYPE_DATAELEMENT) {
+            return (
+                <>
+                    {getIcon(DIMENSION_TYPE_DATA_ELEMENT)}
+                    <span className="data-element-label">{label}</span>
+                    <style jsx>{styles}</style>
+                </>
+            )
+        }
+
+        return (
+            <>
+                <span className="operator-label">{label}</span>
+                <style jsx>{styles}</style>
+            </>
+        )
+    }
+
+    return (
+        <>
+            <div
+                ref={setNodeRef}
+                {...attributes}
+                {...listeners}
+                className={isLast && 'last-item'}
```

Comment:
this works, but I'd still suggest using `cx` for consistency
```suggestion
                className={cx({ 'last-item': isLast })}
```